docs/architecture-decision-record/ADR-0008 - Choice of test framework for intradomain integration points when testing prior to dev.MD

```diff
@@ -0,0 +1,87 @@
+It is assumed that the test will be initiated by placing valid test data on an EventHub from which Databricks will fetch the time series.
+(No seed data required)
+
+* ### Integration to an Sql database
+
+Read/Write access for reading and storing a charge from an azure functions.
+The database must include seed data (MarketParticipants table), Charge test data is not required.
+
+## Considered Options
+
+* Option 1: Running integration tests in a Test Container
+* Option 2: Running integration tests in a PR environment
+* Option 3: Using a test framework like Squadron, that enables to test in a container setup with the ability to connect to services in Azure.
+
+* … <!-- numbers of options can vary -->
+
+## Decision Outcome
+
+For use cases regarding components like eventhub, service bus and Sql database we have implemented a Sqaudron based solution, where it is possible both local-hosted and in CI pipeline to ensure application-to-service  connections.
+
+***Regarding the use case for DeltaLake - it has not yet been finalized on which solution is most valuable, and therefore the decision to cover this is still pending.***
+
+### Positive Consequences <!-- optional -->
+
+* Besides testing component connections locally Squadron allows verification that components are connecting correctly to storage instances by creating a containerized setup where both application layer and service is connected in an infrastructure setup (excluding Terraform - further outline on this under negative consequences).
+
+* The solution is implemented to run both locally and in CI setup.
+
+* This adds to  even more risk elements to be covered prior to deployment, and mitigiates risks that essentially could cause instability and issues on environments and further out in the CD pipeline, which would enhance the complete lead time for functionality delivery.
+
+### Negative Consequences <!-- optional -->
+
+One risk area that has been mentioned for this strategy for testing prior to dev is that the solution does not cover testing of compliance and quality of the solution deployed with correct Terraform modules. This risk is also be present in a test container setup - but could be mitigated by a PR environment solution. We are already using a PR environment solution to ensure test on Terraform and infrastructure - and it was also tested as a solution for intra-domain integration testing. This is can be done, but depending on the testing discipline a very expensive solution, and is therefore not the approach we have choosed.
+
+Given the isolated testing on components and infrastructure, we are mitigating the risk of this issue, by having the possibility of testing services in the PR environment - but given price plans etc the PR environment is triggered in PR gate and torn down immedialtely after given tests have been performed, and should not be used for debugging in the PR gate over a longer period of time.
```

Comment:
Of course it is possible to run tests on the PR environment, but I believe we must completely get away from this approach? Anyways if the PR environment is torn down immediately testing on it won't be possible

Reply:
could we agree that line 83 is then removed totally?

Reply:
Done

Reply:
removed